src/utils/rows-api/report.ts

```diff
@@ -0,0 +1,45 @@
+import { getCurrentTab } from '../chrome';
+import fetch from './fetch';
+
+const TABLE_HEADERS = [
+  'UTC datetime',
+  'URL',
+  'Domain',
+  'Browser',
+  'Browser version',
+  'Report type'
+];
+
+const SHEET_ID = '1aL6dsP2jqENGlkSq3A5O9';
+const TABLE_ID = '87fa52ac-40fc-4d3f-9aff-0cb98f0e9b0d';
+
+export async function createReportHeader() {
```

Comment:
IMHO, you can leave this out and assume that the spreadsheet is already created and it contains the headers as well.

Reply:
Ok, that makes sense, I will remove it.